.changeset/stale-pianos-teach.md

```diff
@@ -0,0 +1,6 @@
+---
+"stylelint": patch
```

Comment:
```suggestion
"stylelint": minor
```

ref: https://stylelint.io/about/semantic-versioning/#minor-release
e.g. `a, a, a {}`

Reply:
Is this really reporting **new** errors?

Before this change it would first report for a single `a` and after the user fixes that error it would report for a second `a`.

Reply:
Iv been asked to switch to `minor` for the same reason before.
It depends on your interpretation of

> a bug fix in a rule that results in Stylelint reporting _more errors_

i.e. 2 warnings/errors instead of 1 with my example

Reply:
Sure!

I thought that the intention was that if Stylelint runs without errors and still runs without errors after an update that it implies a patch.

But happy to adjust the changeset.

Reply:
> I thought that the intention was that if Stylelint runs without errors and still runs without errors after an update that it implies a patch.

Probably, yes.

> But happy to adjust the changeset.

I guess this bullet point would be less subject to interpretation if we had a tool like https://github.com/hjylewis/esplint but for Stylelint.